File: Chronos.App/DataContracts/RequestsOfWork/RequestOfWork.cs
namespace Chronos.App.DataContracts.RequestsOfWork
{
    public class RequestOfWork
    {
        public int RequestOfWorkId { get; set; }
        public string ProductName { get; set; }
        public string Name { get; set; }
        public string FullNumber { get; set; }
        public int? Priority { get; set; }
        public RequestOfWorkStatus StatusId { get; set; }
        public string StatusName { get; set; }
    }
}

File: Chronos.App/ClientApp/src/app/models/request-of-work/request-of-work.ts
import { RequestOfWorkStatus } from "src/app/enums";

export class RequestOfWork {
    requestOfWorkId = 0;
    productName = '';
    name = '';
    fullNumber = '';
    priority?: number = null;
    status: RequestOfWorkStatus;
    statusName = '';

    constructor(obj?: Partial<RequestOfWork>) {
        if (obj)
            Object.assign(this, obj);
    }
}
